search-sentiments-map/public/script.js

```diff
@@ -75,3 +75,25 @@ function showMoreOrLess() {
     showMoreOrLessToggleItem.innerText = TOGGLE_SHOW_MORE;
   }
 }
+
+/** 
+ *  Gets the sentiment score of an inputted search topic. 
+ *  TODO(ntarn): Get the average sentiment score for the inputted search topic's search results. 
+ */
+function getSentiment() {
+  const searchTopic = document.getElementById('search-topic').value;
+  const sentimentScoreElement = document.getElementById('sentiment-score');
+  const searchTopicObject = "searchTopic=" + encodeURIComponent(searchTopic);
+  fetch('/sentiment', {method: 'POST',  // Send a request to the URL.
+    headers: new Headers({
+      'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
What is x-www-form-urlencoded? Why do you need this instead of plain text?

Reply:
it is url encoded (the format)